py/repl.c

```diff
@@ -59,12 +59,7 @@ bool mp_repl_continue_with_input(const char *input) {
         || str_startswith_word(input, "class")
         ;
 
-    // check for unmatched open bracket or quote
-    #define Q_NONE (0)
-    #define Q_1_SINGLE (1)
-    #define Q_1_DOUBLE (2)
-    #define Q_3_SINGLE (3)
-    #define Q_3_DOUBLE (4)
```

Comment:
You should keep these defines in the .c file.  There's no reason to polute the namespace of someone importing repl.h.